lib/cb/requests/job_search/get.rb

```diff
@@ -29,7 +29,7 @@ def http_method
 
         def headers
           {
-            'HostSite' => args[:host_site] || Cb.configuration.host_site,
```

Comment:
I can't really say.  I added back this feature along with the new feature for now.